src/pages/marketplace/generative/[id].tsx

```diff
@@ -317,6 +319,7 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
   return {
     props: {
       token: token,
+      urlQuery,
```

Comment:
We should only return the query parameter we want to keep the seralised payload as small as possible

Reply:
As per the direction of the comments, I made changes and push my code.

Questions:
1. I added "prefereredMarketplaceSorting" boolean field on ISettingsProperties interface. What to do next? Will it be functional later?

Reply:
Sorry but I cant provide a step by step guidance. you have to checkout how the setting context works and store the setting there.